lblod/frontend-subsidiedatabank
# feature: add readonly + cleanup + missing shamecss

## ID
 DGS-81

 ## Description

Make every subsidy form readonly, I did this with the 'show' flag. I also added missing shame.css for the table styling + some cleanup. Lastly I removed the 'last-modifier' field in the headers since we don't need it atm.

You'll also see me removing the `au-c-form--disabled` css class which didn't do anything, it's a possible alternative to make forms readonly but is incomplete atm (see https://github.com/lblod/frontend-loket/blob/development/app/styles/_shame.scss#L11).

 ## Type of change



 ## How to test

Visit the subsidiedatabank frontend and open a subsidy. Try different steps and verify that the forms are readonly.
